src/query_ctrl.ts

```diff
@@ -127,6 +128,14 @@ export class DruidQueryCtrl extends QueryCtrl {
         .then(callback);
     };
 
+    this.getFilterValues = (query, callback) => {
+      let dimension = this.target.currentFilter.dimension;
+      this.datasource.getFilterValues(this.target, this.panelCtrl.range, query)
```

Comment:
not sure if we should use last 24 hours or the date range from panel to fulfill the request. As of not using panel time range